db/migrate/20170602144526_add_additional_info_to_booking_request.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdditionalInfoToBookingRequest < ActiveRecord::Migration[5.1]
+  def change
+    add_column :booking_requests, :additional_info, :string
```

Comment:
These ought to be combined into a single migration. Also they should not be nullable and they should default to an empty string.